models/basic_block.py

```diff
@@ -21,31 +25,97 @@ def block(f=64, k=3, s=2, order=None, order_param=None, order_priority=False):
     :param order_param: type dict of dict: Parameter of :param order
     :param order_priority : type bool: True if bypassing  :param f, :param k, and :param s  , which are already
                                         determined by convolution parameter in :param  order_param
+    **kwargs: supported parameters: name
     :return: func: Model function
     """
 
     if order is None:
         order = ['c', 'r', 'b']
+
+    #print("len order param: ", len(order_param))
+    #print("len order: ", len(order))
+    #assert (len(order) == len(order_param))
+
+    order_param_items = []
+    activation_func = ['l', 'e']
+    normalizations = ['g', 'b']
+    pooling = ['mp', 'ap']
+    for elem in order:
```

Comment:
backward comp.?

Reply:
Lines of code deleted. I used them because there was a problem while testing the network, but now it is okay if the lines are not there.